setup.py

```diff
@@ -5,6 +5,7 @@
 
 base = os.path.dirname(os.path.abspath(__file__))
 README_PATH = os.path.join(base, "README.rst")
+TORNADO_VERSION = os.environ.get("TORNADO_VERSION", "")
```

Comment:
Since the setup.py is used in transitive dependency declarations, it would be better if the definition reflected the range of all versions that tornado-transmute supports. So I guess at this point, it theoretically supports all of them, so we should remove the specifier altogether.

If you're looking for a way to ensure that we can test against multiple versions, could we use that environment variable in the main method of uranium?: https://github.com/toumorokoshi/tornado-transmute/blob/master/ubuild.py#L6